spec/workers/pages/bust_cache_worker_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+class NewPageBuster
+  def self.bust_page(*); end
+end
+
+RSpec.describe Pages::BustCacheWorker, type: :worker do
+  include_examples "#enqueues_on_correct_queue", "high_priority", 1
+
+  describe "#perform" do
+    let(:worker) { subject }
+    let(:page_slug) { "test123" }
+
+    it "with empty slug does not call the cache buster" do
+      allow(CacheBuster).to receive(:bust_page)
+      worker.perform("")
+      expect(CacheBuster).not_to have_received(:bust_page)
+    end
+
+    it "with cache buster defined busts cache with defined buster" do
```

Comment:
Feeling like these are good candidates for another shared example. I just want to wait for the third strike so there is a clearer established pattern before we refactor :)

Reply:
I'd wait for the end of the sprint TBH, coordinating such refactors with contributions between possibly dozens of contributors is quite a feat. Let the patterns emerge, then we can fix them after :)